.gitignore

```diff
@@ -15,4 +15,6 @@ working*
 
 App.config.mine
 src/packages/*
-!src/packages/repositories.config
\ No newline at end of file
+!src/packages/repositories.config
+src/.vs/config/applicationhost.config
```

Comment:
I don't know why it got added. You can probably change it to ignore that folder after the checkin.